Troubleshoot/ihp-troubleshoot

```diff
@@ -162,6 +163,21 @@ with open(path.join(Path.home(), ".config/nix/nix.conf")) as file:
     else:
         show_success("No legacy cachix key found")
 
+show_step("Validating and repairing /nix/store/ (could take a minute)")
+
+_, nix_store_troubleshooting_output = run_command(
+    ["nix-store", "--verify",
+     "--check-contents", "--repair",
+     "--quiet", "--no-build-output"]
+)
+
+if "warning: not all errors were fixed" in nix_store_troubleshooting_output:
+    show_failure(
+        "Errors where found in /nix/store/, run `rm -rf build; nix-shell --run 'make build/ihp-lib'`"
```

Comment:
Is this the correct command needed to be run if IHP packages are corrupt in `/nix/store/`?

Reply:
I'd leave this `nix-store --verify` step out for now. It's super slow because it has to recheck the whole nix store and also it has never happend to an IHP user yet